lighthouse-core/test/audits/cache-start-url.js

```diff
@@ -43,18 +37,30 @@ describe('Cache: start_url audit', () => {
     return assert.equal(Audit.audit({Manifest, CacheContents}).rawValue, false);
   });
 
-  // Need to disable camelcase check for dealing with short_name.
+  // Need to disable camelcase check for dealing with start_url.
   /* eslint-disable camelcase */
-  it('fails when a manifest contains no start_url', () => {
+  it('fails when a manifest artifact contains no start_url', () => {
+    const inputs = {
+      Manifest: { }
+    };
+    return assert.equal(Audit.audit(inputs).rawValue, false);
+  });
+
+  it('fails when a manifest artifact contains a null start_url', () => {
     const inputs = {
       Manifest: {
         start_url: null
       }
     };
-
     return assert.equal(Audit.audit(inputs).rawValue, false);
   });
 
+  it('fails when a manifest contains no start_url', () => {
+    const inputs = {
+      Manifest: manifestParser({})
```

Comment:
Will you switch the two tests above to use the parser too? They may be redundant with this one at that point, not sure.